ironfish/src/testUtilities/fixtures.ts

```diff
@@ -223,6 +223,29 @@ export async function useMinerBlockFixture(
   )
 }
 
+export async function useMintBlockFixture(options: {
```

Comment:
With this fixture and `usePostTxFixture` we'll have added a couple that use an options object instead of a list of parameters. Should we make a point of updating the other fixtures to use `options` for consistency?

Reply:
yes, im gonna do a full pass on that and clean up all the fixtures in one PR soon.